lib/liveValidation/operationSearcher.ts

```diff
@@ -87,7 +87,7 @@ export class OperationSearcher {
         if (!apiVersion) {
           this.logging(
             `Unable to find apiVersion for path : "${pathObject._pathTemplate}".`,
-            LiveValidatorLoggingLevels.error,
+            LiveValidatorLoggingLevels.warn,
```

Comment:
Can you revert this loglevel change either?

Reply:
reverted